src/routes/about/+page.svelte

```diff
@@ -1,14 +1,22 @@
 <script lang="ts">
+	import { run } from 'svelte/legacy';
+
 	import { version } from '$app/environment';
 	import { fade } from 'svelte/transition';
 	import type { PageData } from './$types';
 	import { goto } from '$app/navigation';
 	import { DISCORD_SERVER_INVITE_URL } from '../../const';
 
-	export let data: PageData;
+	interface Props {
+		data: PageData;
+	}
+
+	let { data }: Props = $props();
 
-	let { supabase } = data;
-	$: ({ supabase } = data);
+	let { supabase } = $state(data);
+	run(() => {
```

Comment:
same here? surely there's a way to handle this for supabase's sake LOL

Reply:
```js
let { supabase } = $derived(data);
```